arm-sql/compositeSql.json

```diff
@@ -0,0 +1,23 @@
+{
+  "info": {
+    "title": "SqlManagementClient",
+    "description": "Composite Swagger for Azure Sql Database Client"
+  },
+  "documents": [
+    //"./2015-05-01/swagger/jobaccount.json",
```

Comment:
Minor - This file is having reference to all other files, which are not in this review now